src/site/content/en/blog/ecommerce-notifications-best-practice/index.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Use best practices for e-commerce notifications
+subhead: Push notifications can help you engage with your customers. Here's how to do them right. 
+authors:
+  - katejeffreys
+date: 2019-07-30
+hero: hero.jpg
+alt: A man looking at his phone at sunset. Photo by @brunoreyna on unsplash.com.
+description: |
+  Learn best practices for implementing push notifications in an e-commerce web app.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - notifications
+  - push
+  - e-commerce
+---
+
+When used effectively, push notifications help your customers to get the most
```

Comment:
Nit: 'help your customers get the most' sounds better to me than 'help your customers to get the most'.